docs/INSTALLATION.md

```diff
@@ -1,17 +1,51 @@
 # Installation
 
+# What awk do I have?
+
+The Exercism AWK track uses `gawk`, GNU's AWK.
+You may already have it, let's see: execute this command
+```sh
+awk --version
+```
+
+If that returns:
+
+* something like `GNU Awk 5.1.0 ...`
+    - You've already got GNU awk, you can carry on with the exercises.
+
+* something like `awk version 20200816`
+    - This is is the "one true awk" implementation.
```

Comment:
Is this BSD awk?

Reply:
I think so. I don't have one to test on except my Mac.

Reply:
I don't have Mac either, but I'm pretty sure it's BSD.

Reply:
```suggestion
    - This is BSD awk.
```